tests/cephadm/test_admin_node_removal.py

```diff
@@ -0,0 +1,20 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+
+    config = kw.get("config")
+    # Removing _admin node from bootstrp node
+    instance = CephAdmin(cluster=ceph_cluster, **config)
+    admin_node = instance.cluster.get_nodes()[0].hostname
+    cmd = "cephadm shell ceph orch host rm {}".format(admin_node)
+    try:
+        out, err = instance.installer.exec_command(cmd, sudo=True)
+        log.error("Test case failed, Admin node removed")
```

Comment:
Raise specific exception here

Reply:
As discussed, Added "CommandFailed" exception.

Reply:
As discussed with team and polarion test case owner
While running the "host remove" command as expected command failed with any specific error.
As of now cephci "CommandFailed" exception can be handled by "CommandFailed" exception.